config/lists/events.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" ?>
+<list xmlns="http://schemas.sulu.io/list-builder/list">
+    <key>events</key>
+
+    <properties>
```

Comment:
A list configuration can include fields of associated entities by defining `<join>` tags. An example can be found in the [list configuration of the built-in contact entity](https://github.com/sulu/sulu/blob/2.x/src/Sulu/Bundle/ContactBundle/Resources/config/lists/contacts.xml).

Reply:
fdsa